include/types/Profiles.h

```diff
@@ -38,14 +40,21 @@ class Profiles : public PerformanceTable {
    */
   Profiles(const Profiles &profiles);
 
-  ~Profiles();
+  ~Profiles(){};
```

Comment:
Same comment as for the initialize profile, shouldn't be the implementation only in cpp file?